server/sonar-server/src/test/java/org/sonar/server/permission/ws/template/BulkApplyTemplateActionTest.java

```diff
@@ -104,7 +106,10 @@ public void bulk_apply_template_by_template_uuid() throws Exception {
     ComponentDto view = db.components().insertView(organization);
     loginAsAdmin(organization);
 
-    newRequest().setParam(PARAM_TEMPLATE_ID, template1.getUuid()).execute();
+    newRequest()
+      .setParam(PARAM_TEMPLATE_ID, template1.getUuid())
+      .setParam(PARAM_QUALIFIERS, String.join(",", Qualifiers.PROJECT, Qualifiers.VIEW))
```

Comment:
If default value is added to the qualifiers parameter, this line becomes useless

Reply:
I don't understand. The default value is set if there is no value provided

Reply:
So this line can be removed